### PR TITLE
add readonly to duckdb_databases()

### PR DESCRIPTION
**Pull request by stephaniewang526**
Have some free time. Fixes https://github.com/duckdblabs/motherduck/issues/72

### CODE REVIEW

File: src/function/table/system/duckdb_databases.cpp [hunk 1]
```diff
@@ -29,6 +29,9 @@ static unique_ptr<FunctionData> DuckDBDatabasesBind(ClientContext &context, Tabl
 	names.emplace_back("internal");
 	return_types.emplace_back(LogicalType::BOOLEAN);
 
+	names.emplace_back("readonly");
```
Comment (Mytherin) [inline src/function/table/system/duckdb_databases.cpp:32]: Could we add it as the last element instead?

**Review by Mytherin — COMMENTED** [review]
Thanks for the PR! LGTM - one minor comment

File: src/function/table/system/duckdb_databases.cpp (hunk 1 above)
Comment (stephaniewang526) [reply src/function/table/system/duckdb_databases.cpp]: Done.

**Comment by Mytherin** [comment]
Thanks! LGTM

**Pull request merged**